### PR TITLE
Cran fixes

### PR DESCRIPTION
**Pull request by onurfiliz**
A host of changes to get ready for CRAN submission. I tried to keep each conceptual changeset as a separate commit, so I won't try to summarize everything here.

### CODE REVIEW

**Comment by onurfiliz** [comment]
@cosinequanon please do not merge yet even if you are OK with all the changes. Just a comment to that effect would be sufficient. It is possible that there will be further changes required by the submission, which I will make in this branch.

File: README.md [hunk 1]
```diff
@@ -27,10 +29,24 @@ con <- dbConnect(
 )
 
 res <- dbSendQuery(con, 'SELECT 1')
-# n != -1 for dbFetch is not supported
-data <- dbFetch(res)
+# dbFetch without arguments only returns the current chunk, so we need to
+# loop until the query completes.
+while (!dbHasCompleted(res)) {
+    chunk <- dbFetch(res)
+    print(chunk)
```
Comment (cosinequanon) [inline README.md]: Why are we printing?

Comment (cosinequanon) [inline README.md]: Oh, this is just the README, nvm

File: tests/testthat/test-dplyr.integration.R [hunk 2]
```diff
@@ -22,25 +18,39 @@ test_that('dplyr integration works', {
 
   iris_presto <- dplyr::tbl(db, tablename)
 
-  expect_that(iris_presto %>% head(5) %>% nrow, equals(5))
+  expect_that(nrow(head(iris_presto, 5)), equals(5))
```
Comment (cosinequanon) [inline tests/testthat/test-dplyr.integration.R]: It sucks we have can't use the `%>%` operator :( oh well

File: tests/testthat/test-data_types.R [hunk 3]
```diff
@@ -136,7 +136,7 @@ test_that("all data types work", {
     type_array_bigint=NA,
     type_map_varchar_bigint=NA
   )
-  attr(e[['type_timestamp_with_timezone']], 'tzone') <- NA_character_
+  attr(e[['type_timestamp_with_timezone']], 'tzone') <- NULL
```
Comment (cosinequanon) [inline tests/testthat/test-data_types.R]: I'm sure you had good reason but I'm curious, why did this need changing?

**Comment by cosinequanon** [comment]
slowclap

This is amazing, thank you so much for cleaning this stuff up. Feel free to merge whenever.

File: tests/testthat/test-dplyr.integration.R (hunk 2 above)
Comment (onurfiliz) [inline tests/testthat/test-dplyr.integration.R]: Yeah, makes you appreciate how much nicer things are _with_ it.

File: tests/testthat/test-data_types.R (hunk 3 above)
Comment (onurfiliz) [inline tests/testthat/test-data_types.R]: See the [commit message](https://github.com/prestodb/RPresto/pull/48/commits/301ce60b653a99617d3e84cd67c2da84821c36bd) for the corresponding commit.

Two reasons:
- R 3.1.x complains with NA timezone.
- Let's say you have a 0-row with US/Pacific and 1-row with US/Eastern. dplyr::bind_rows() switches to UTC when they differ.

**Pull request merged**